lms/djangoapps/grades/tests/test_new.py

```diff
@@ -386,7 +386,7 @@ def test_score_submission_for_all_problems(self):
             # Configure one block to return no possible score, the rest to return 3.0 earned / 7.0 possible
             block_count = self.SCORED_BLOCK_COUNT - 1
             mock_score.side_effect = itertools.chain(
-                [(earned_per_block, None, earned_per_block, None, True)],
+                [(earned_per_block, None, earned_per_block, None, True, datetime.datetime(2000, 1, 1))],
```

Comment:
I just noticed the `TODO TNL-5930` in this file.  Seems like an appropriate time to address it.